# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [19]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [29]:
s1 = pd.Series(np.random.randint(1,4, 100))
s2 = pd.Series(np.random.randint(1,3, 100))
s3 = pd.Series(np.random.randint(10000, 30000, 100))

### Step 3. Let's create a DataFrame by joinning the Series by column

In [30]:
house = pd.DataFrame({"bedrs": s1, "bathrs": s2, "price_sqr_meter": s3})
house

,bedrs,bathrs,price_sqr_meter
0,1,1,13586
1,1,1,26192
2,2,2,28868
3,3,1,13411
4,2,1,13984
...,...,...,...
95,1,2,10886
96,2,1,18545
97,3,2,14002
98,3,2,12180


### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [31]:
house.head()

,bedrs,bathrs,price_sqr_meter
0,1,1,13586
1,1,1,26192
2,2,2,28868
3,3,1,13411
4,2,1,13984


### Step 5. Create a column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [ ]:
bigcolumn = pd.concat([s1,s2,s3])
bigcolumn

0         1
1         1
2         2
3         3
4         2
      ...  
95    10886
96    18545
97    14002
98    12180
99    24800
Length: 300, dtype: int32

In [35]:
bigcolumn.to_frame()

,0
0,1
1,1
2,2
3,3
4,2
...,...
95,10886
96,18545
97,14002
98,12180


### Step 6. Oops, it seems it is going only until index 99. Is it true?

In [ ]:
bigcolumn = pd.concat([s1,s2,s3], ignore_index=True)

In [37]:
bigcolumn

0          1
1          1
2          2
3          3
4          2
       ...  
295    10886
296    18545
297    14002
298    12180
299    24800
Length: 300, dtype: int32

### Step 7. Reindex the DataFrame so it goes from 0 to 299

In [39]:
bigcolumn.to_frame()

,0
0,1
1,1
2,2
3,3
4,2
...,...
295,10886
296,18545
297,14002
298,12180
